ext/nnstreamer/tensor_filter/tensor_filter_tensorflow_core.cc

```diff
@@ -350,112 +371,35 @@ TFCore::run (const GstTensorMemory * input, GstTensorMemory * output)
 #endif
 
   std::vector <std::pair <string, Tensor>> input_feeds;
-  std::vector <string> output_tensor_names;
   std::vector <Tensor> outputs;
-  int array_len;
 
   for (int i = 0; i < inputTensorMeta.num_tensors; ++i) {
-    TensorShape ts = TensorShape ({});
-    for (int j = inputTensorRank[i] - 1; j >= 0; j--) {
-      ts.AddDim (inputTensorMeta.info[i].dimension[j]);
-    }
-    Tensor inputTensor (
-      getTensorTypeToTF (input[i].type),
-      ts
-    );
-
-    array_len = input[i].size / tensor_element_size[input[i].type];
-
-    switch (input[i].type) {
-      case _NNS_INT32:
-        copyInputWithType (int32);
-        break;
-      case _NNS_UINT32:
-        copyInputWithType (uint32);
-        break;
-      case _NNS_INT16:
-        copyInputWithType (int16);
-        break;
-      case _NNS_UINT16:
-        copyInputWithType (uint16);
-        break;
-      case _NNS_INT8:
-        copyInputWithType (int8);
-        break;
-      case _NNS_UINT8:
-        copyInputWithType (uint8);
-        break;
-      case _NNS_INT64:
-        copyInputWithType (int64);
-        break;
-      case _NNS_UINT64:
-        copyInputWithType (uint64);
-        break;
-      case _NNS_FLOAT32:
-        copyInputWithType (float);
-        break;
-      case _NNS_FLOAT64:
-        copyInputWithType (double);
-        break;
-      default:
-        /** @todo Support other types */
-        break;
-    }
+    Tensor in (input_tensor_info[i].type, input_tensor_info[i].shape);
```

Comment:
I will revive `TensorShape` when the `TensorCApi::makeTensor()` is available in our system.